javascript/computer_science/hash_map_data_structure.md

```diff
@@ -88,11 +88,17 @@ Buckets are storage that we need to store our elements. Simply, it's an array. F
 
 1. Pass "Fred" into the hash function to get the hash code which is `508`.
 1. Find the bucket at index `508`.
-1. Store the key value pair in that bucket.
+1. Store the key value pair in that bucket, in this case the value would be `undefined` as we haven't specified it.
```

Comment:
I think we could also change this section to mention a value to store. This would probably be less confusing? Do you have any thoughts @nikitarevenco ?

Reply:
Yes, I think that's a good idea. What do you think of this:

Let's say we wanted to store a person's full name as a key "Fred" with a value of "Smith":

1. Pass "Fred" into the hash function to get the hash code which is `508`.
1. Find the bucket at index `508`.
1. Store the key value pair in that bucket. In this case, the key would be `508` and the value would be "Smith".

Reply:
Alright, I've implemented it.